dotcom-rendering/src/model/enhance-ad-placeholders.test.ts

```diff
@@ -28,7 +28,7 @@ const getImageElement = (): ImageBlockElement => ({
 	media: { allImages: [] },
 	data: {},
 	displayCredit: true,
-	role: 'supporting',
+	role: 'inline',
```

Comment:
aiui making role 'inline' makes the image eligible to add an ad placeholder _after_ the image but this element is used by the 'should not insert an ad placeholder before an image element'.

Do we need separate image before and after tests to make this clearer?

Reply:
Yep you're right - we should expand the tests to include not inserting after a thumbnail/supporting image - will do that!